### DISCUSSION TITLE
Multiselect for Input Schema

### DISCUSSION THREAD

**Discussion by sunny-kim-92** [apify/apify-client-js#525] [General]
Is it possible to setup a select in the input schema that accepts multiple options? I would like to setup a field like country here: https://docs.apify.com/platform/actors/development/actor-definition/input-schema/specification/v1#string. However, I would like users to be able to select multiple entries.

**Comment by B4nan** [comment]
multiselects are described at the very end of the same page https://docs.apify.com/platform/actors/development/actor-definition/input-schema/specification/v1#array

**Reply by sunny-kim-92** [reply]
Thank you for the quick response. I only want to allow users to select from a list of options, as opposed to manual entry, is there a way to do this? I could validate the response using regular expressions, but I was referencing the multiple attribute in a select element.

**Reply by B4nan** [reply]
thats exactly how it works, check the very last example on that page
